save-cloud-charts/save-cloud/templates/s3-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if not .Values.s3.external -}}
+
+  {{- if eq .Values.profile "dev" -}}
```

Comment:
I'm not sure we need this `if` here: if s3 is internal, we should create a secret and service, when it's external -- secret is created in values-cce

Reply:
Makes sense, will remove it

Reply:
Done